appveyor.yml

```diff
@@ -5,7 +5,6 @@ init:
 install:
   - SET PATH=C:\Ruby21-x64\bin;%PATH%
   - SET LOG_SPEC_ORDER=true
-  - gem install bundler --quiet --no-ri --no-rdoc
```

Comment:
Just read the commit. A lot of times this was for the older versions of Ruby - to ensure that it was installed and updated.